src/desktop/WebDialog.ts

```diff
@@ -81,12 +85,18 @@ export class WebDialogController implements IWebDialogController {
 
 		bw.once('ready-to-show', () => bw.show())
 		bw.webContents.on("did-fail-load", () => closeDefer.reject(new Error(`Could not load web dialog at ${urlToOpen}`)))
-		await bw.loadURL(urlToOpen.toString())
 
-		const facade = await this.initRemoteWebauthn<FacadeType>(bw.webContents)
+		// Don't wait for the facade to init here, because that only happens after we call `bw.loadUrl`
```

Comment:
Good catch!

Reply:
Alternatively we could set up listener and send init separately but I think this solution is also good